Incorrect import?

I guess you have incorrect import which was added by IDE auto import functionality. 

[The line](https://github.com/crcn/sift.js/blob/master/src/operations.ts#L16) imports `isArray` from nodejs `util` module but instead should import `isArray` from `./utils`

Angular cli gives error:

> ERROR in ../casl/packages/casl-ability/node_modules/sift/es5m/operations.js
Module not found: Error: Can't resolve 'util' in '/home/sergii/projects/casl/packages/casl-ability/node_modules/sift/es5m'

--- a/src/operations.ts
+++ b/src/operations.ts
@@ -12,8 +12,7 @@ import {
   GroupOperation,
   numericalOperation
 } from "./core";
-import { Key, comparable, isFunction } from "./utils";
-import { isArray } from "util";
+import { Key, comparable, isFunction, isArray } from "./utils";
 
 class $Ne extends BaseOperation<any> {
   private _test: Tester;

